pkg/apis/identity/v0alpha1/types_team.go

```diff
@@ -60,6 +59,21 @@ type TeamRef struct {
 	Name string `json:"name,omitempty"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type TeamMemberList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+
+	Items []TeamMember `json:"items,omitempty"`
+}
+
+type TeamMember struct {
+	IdentityDisplay `json:",inline"`
+
+	// Permission member has in team.
+	Permission TeamPermission `json:"permission,omitempty"`
```

Comment:
IIUC, this is where we would eventually add the reference to the resource that defined the team membership

BUT for now, we should add the 'external' bool property that is filled up from our query

Reply:
Oh yeah totally forgot about that. So then when we are fetching bindings we should exclude all external and here we just set that boolean right. Even though both TeamBindings and this is read only for now that is kinda how it is going to look like / work later